tests/basic/pool_function_test.go

```diff
@@ -186,3 +189,152 @@ var _ = Describe("{PoolExpandWithPXRestart}", func() {
 		})
 	})
 })
+
+var _ = Describe("{PoolExpandResizeInvalidPoolID}", func() {
+
+	BeforeEach(func() {
+		StartTorpedoTest("PoolExpandResizeInvalidPoolID",
+			"Initiate pool expansion using invalid Id", nil, testrailID)
+	})
+
+	AfterEach(func() {
+		EndTorpedoTest()
+	})
+
+	stepLog := "Resize with invalid pool ID"
+	log.InfoD(stepLog)
+	It(stepLog, func() {
+		// pick node to resize
+		nodes := node.GetStorageNodes()
+		nodetoPick := nodes[0]
+
+		// invalidPoolUUID Generation
+		id := uuid.New()
+		invalidPoolUUID := id.String()
+
+		// Resize Pool with Invalid Pool ID
+		stepLog = fmt.Sprintf("Expanding pool on node [%s] using auto", nodetoPick.Name)
+		Step(stepLog, func() {
+			resizeErr := Inst().V.ExpandPool(invalidPoolUUID, api.SdkStoragePool_RESIZE_TYPE_AUTO, 100, true)
+			dash.VerifyFatal(resizeErr != nil, true, "Verify error occurs with invalid Pool UUID")
+			// Verify error on pool expansion failure
+			var errMatch error
+			re := regexp.MustCompile(fmt.Sprintf(".*failed to find storage pool with UID.*%s.*",
```

Comment:
The text of this error message might change in future and we wouldn't want to enforce a strict match. 
Verifying `resizeErr != nil` should suffice, let's not do regex match.

Reply:
lets keep that for better know why its failing for

Reply:
We don't have error code yet, so for now we can do string match in order to know why its exactly failed,
and in future the error message changes i future, i will be his/her task to change the automation aswell

Reply:
I actually like the regex check here since it is a user facing error message. This keeps us in check on what errors we return back and display on the CLI to the user. If there is any change in the error message then this job should fail since that was not the original expectation, and it also enforces them to think twice if such a message needs to be changed and if so they should update the automation too. 

We can also do error code checks, but since we don't have them we don't need to block this effort.